device/dell/x86_64-dellemc_z9264f_c3538-r0/plugins/psuutil.py

```diff
@@ -74,11 +84,10 @@ def get_psu_status(self, index):
         :return: Boolean, True if PSU is operating properly, False if PSU is\
         faulty
         """
-        #Until psu_status is implemented this is hardcoded temporarily
+    # Until psu_status is implemented this is hardcoded temporarily
```

Comment:
Comment should be indented another level, as it's part of the function.

Reply:
Hi @jleveque
     Addressed your latest valuable comments...
Can you please review.
     
Thanks & Regards,
Karthik.G